### ISSUE TITLE
[Bug]: Iron's Spells and Mine and Slash BUG

### ISSUE THREAD

**Issue by Meidde** [RobertSkalko/Mine-And-Slash-Rework#155]
### Describe the bug you're experiencing

![image](https://github.com/user-attachments/assets/236ddb61-7fcc-48e5-814f-73d86de78526)
If you use Iron's spells and mine and slash together, the damage from Ironsspells stops working,

I tried adding soul to the grimoire and staff and even then the spell doesn't work the damage

### Reproducability

Try use irons spells and mine and slash

### Mod up to date

- [X] Before submitting this issue I updated to the newest version and reproduced it

### Tested Without Custom Mine and Slash Datapacks

- [X] In case I used Mine and Slash datapacks, I ran Mine and Slash in a new instance without custom datapacks and confirmed the bug still exists

### Mine and Slash version

Mine_and_Slash-1.20.1-v5.9.30

### Logs

sad

**Comment by RobertSkalko** [comment]
Yes i know, Mns isn't compatible with any other spell mods currently

**Comment by RobertSkalko** [comment]
An addon to enable compatibility is coming, when it's released, i'd like feedback on whether it works good enough

**Comment by RobertSkalko** [comment]
https://www.curseforge.com/minecraft/mc-mods/mine-and-slash-compatibility

This addon will now switch mns into a "lite" version which should be more compatible with other mods, at the expense of any sense of balance.

Tell me if it works and if you like it

**Comment by RobertSkalko** [comment]
Also added a new feature to convert iron spells and other spell mod damage to mns damage if you install the compat addon.
Could use some feedback though

**Comment by RobertSkalko** [comment]
Ok seeing as nobody is responding, closing this.

If you want Mine and slash to work with other spell mods, install: https://www.curseforge.com/minecraft/mc-mods/mine-and-slash-compatibility

And set the mine and slash compatibility toml config to COMPATIBLE
